build.gradle.kts

```diff
@@ -16,7 +16,8 @@ val prometheusVersion = "0.16.0"
 val junitJupiterVersion = "5.10.1"
 val verapdfVersion = "1.24.1"
 val ktfmtVersion = "0.44"
-val kotlinloggerVesion = "5.1.0"
+val kotlinloggerVersion = "5.1.0"
+val kotestVersion = "5.8.0"
```

Comment:
Trenger vi kotest her, trodde bare junit ble brukt?

Reply:
Hadde sikkert holdt med junit som du sier. Skal fjerne avhengighet

Reply:
Kotest bruker Junit men gir bare ekstra verktøy for validering av testresultater osv. Så kan jo diskuteres om det er nyttig eller ikke. Synes det gjør testene mer lesbart

Reply:
Men kan godt fjerne det helt til det blir større behov eventuelt